Src/IronPython/Compiler/Ast/Comprehension.cs

```diff
@@ -262,7 +262,7 @@ internal class ComprehensionScope : ScopeStatement {
             return false;
         }
 
-        internal override PythonVariable BindReference(PythonNameBinder binder, PythonReference reference) {
+       internal override PythonVariable BindReference(PythonNameBinder binder, PythonReference reference) {
```

Comment:
Fix the change in indent?
```suggestion
        internal override PythonVariable BindReference(PythonNameBinder binder, PythonReference reference) {
```